R/utils.R

```diff
@@ -587,10 +590,64 @@ wb_dims <- function(..., select = NULL) {
     stop("Internal error, all arguments should be named after this point.")
   }
 
+  # handle from_dims
+
+  if (!is.null(args$from_dims)) {
+    if (!is.null(args$from_col) || !is.null(args$from_row)) {
+      stop("Can't handle `from_row` and `from_col` if `from_dims` is supplied.")
+    }
+    # transform to
+    from_row_and_col <- dims_to_rowcol(args$from_dims, as_integer = TRUE)
+
+    left  <- args$left
+    right <- args$right
+    above <- args$above
+    below <- args$below
+
+    from_col <- col2int(from_row_and_col[[1]])
+    from_row <- as.integer(from_row_and_col[[2]])
+
+    # can only be one
+    if (length(c(left, right, above, below)) > 1)
+      stop("can only be one direction")
+
+    if (!is.null(left)) {
+      fcol <- min(from_col) - left - ncol(args$x) + 1L
+      frow <- min(from_row)
+    } else if (!is.null(right)) {
+      fcol <- max(from_col) + right
+      frow <- min(from_row)
+    } else if (!is.null(above)) {
+      fcol <- min(from_col)
+      frow <- min(from_row) - above - nrow(args$x)
```

Comment:
this might require a column name correction

Reply:
Oh no, this is hellish a bit. Took me a while to figure it properly without these.

Reply:
Hehe, yeah, you've created a complex monster with `wb_dims()`. I have implemented something, maybe have a look if you have time. I probably wont have much time to work on this the upcoming week. Was thinking about corner cases, because ... why not `wb_dims(from_dims = "A2:B3", below = 4, right = 5)`. But we can wait until anyone asks for it. Probably going with what we have and documenting it a bit should be enough for this PR.